src/containers/Article.js

```diff
@@ -486,15 +446,94 @@ Article.propTypes = {
   entities: PropTypes.object,
   match: PropTypes.object,
   selectedPost: PropTypes.object,
-  theme: constPropTypes.theme
+  styles: PropTypes.shape({
+    text: PropTypes.shape({
+      fontColor: PropTypes.string
+    }),
+    title: PropTypes.shape({
+      fontColor: PropTypes.string
+    }),
+    subtitle: PropTypes.shape({
+      fontColor: PropTypes.string
+    }),
+    topic: PropTypes.shape({
+      fontColor: PropTypes.string
+    })
+  }).isRequired,
+  isLeadingAssetFullScreen: PropTypes.bool.isRequired
 }
 
 Article.defaultProps = {
   entities: {},
   // react-router `match` object
   match: {},
-  selectedPost: {},
-  theme: constPageThemes.defaultTheme
+  selectedPost: {}
+}
+
+function chooseStyles(articleStyle) {
```

Comment:
Will it be better to place function `chooseStyles` and `isLeadingAssetFullScreen` in `src/managers/theme-manager.js`?

Reply:
About this advice, I created [an issue](https://github.com/twreporter/twreporter-react/issues/1118) for tracking.